### PR TITLE
allow julia path relative to atom executable

### PR DESCRIPTION
**Pull request by pfitzseb**

### CODE REVIEW

File: lib/misc/paths.coffee [hunk 1]
```diff
@@ -17,17 +17,30 @@ module.exports =
...
+      current_dir = process.cwd()
+      exepath = path.dirname(process.execPath)
+      
+      try
+        process.chdir(exepath)
+        realpath = fs.realpathSync(p)
+        if fs.existsSync(realpath) then resolve(realpath)
+      catch err
+        try
+          process.chdir(current_dir)
```
Comment (aviatesk) [inline lib/misc/paths.coffee]: Why isn't this in `finally` block ?

Comment (pfitzseb) [reply lib/misc/paths.coffee]: It is now ;) Good catch.

**Comment by pfitzseb** [comment]
That's useful to have for portable installations and worked up until Juno 0.11 because of the powershell wrapper (and it got requested by @RoyiAvital).

**Comment by aviatesk** [comment]
makes sense 😊👍

**Comment by jkaet** [comment]
Looks like this issue returned with decaffination?

**Comment by pfitzseb** [comment]
I don't think so. We just never made a release that included this fix.

**Pull request merged**